CanalTP/Chaos
# Add new /drafts/disruptions endpoint into swagger

# Description

This PR adds new `/drafts/disruptions` endpoint documentation into swagger

Issue link: BOT-3342

This is new feature

## How to test

Here are the following steps to test this pull request:

Use swagger editor at https://editor.swagger.io/

You should also check in SLATE mode https://canaltp.github.io/Chaos/
